Upgrade to latest upstream release 0.7.22

saraedum opened the pull request.

isuruf committed:
Upload 0.7.16

isuruf committed:
Merge pull request #8 from isuruf/0.7.16

Upload 0.7.16

isuruf committed:
Add patch for segfault

isuruf committed:
Merge pull request #9 from isuruf/0.7.16

Add patch for segfault

saraedum committed:
Upgrade to 0.7.22

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

saraedum commented:
@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I rerendered the feedstock and it seems to be already up-to-date.

isuruf commented on recipe/meta.yaml:
```diff
@@ -8,10 +8,13 @@ package:
-  sha256: d380312bb6157fe2c0f22bf7af0c5ba1a001ae4ddfaf186e1a240ebfb0ac492e
+  sha256: 7fa8e2a8be77c2a64ac794a5ccb53dcbd232f8a2cccad31b876e90f72fcdc090
+  patches:
+    - memleak.patch
```
I think this can be dropped now.

saraedum committed:
drop upstreamed patch

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

isuruf commented:
Can you skip python 3.5 since sage skips 3.5?

saraedum commented:
I don't understand. Why should I skip a version here just because Sage does?

isuruf commented:
Last I checked pynac was tied to sage and was not usable outside sage. Has that changed?

saraedum commented:
I always thought that you could use pynac standalone but it seems you are right: `Pynac is part of Sage and
requires a full Sage installation`

saraedum committed:
Skip Python 3.5

pynac is tied to Sage, so it does not make sense to build it for versions that
sagelib does not build for.

Also I updated the description to make it a bit more clear that this is closely tied to SageMath. (and does not work without.)
